docs/DeveloperGuide.md

```diff
@@ -724,29 +724,47 @@ testers are expected to do more *exploratory* testing.
    1. Re-launch the app by double-clicking the jar file.<br>
        Expected: The most recent window size and location is retained.
 
-1. _{ more test cases …​ }_
+### Deleting an animal
 
-### Deleting a person
+1. Deleting an animal while all animals are being shown
 
-1. Deleting a person while all persons are being shown
+   1. Prerequisites: List all animals using the `list` command. Multiple animals in the list.
 
-   1. Prerequisites: List all persons using the `list` command. Multiple persons in the list.
-
-   1. Test case: `delete 1`<br>
-      Expected: First contact is deleted from the list. Details of the deleted contact shown in the status message. Timestamp in the status bar is updated.
+   1. Test case: `delete 123`<br>
+      Expected: The animal with an ID of 123 deleted from the list. Details of the deleted animal shown in the status message. Timestamp in the status bar is updated.
 
    1. Test case: `delete 0`<br>
-      Expected: No person is deleted. Error details shown in the status message. Status bar remains the same.
+      Expected: No animal is deleted. Error details shown in the status message. Status bar remains the same.
 
-   1. Other incorrect delete commands to try: `delete`, `delete x`, `...` (where x is larger than the list size)<br>
+   1. Other incorrect delete commands to try: `delete`, `delete x`, `...` (where x is smaller than the requisite size)<br>
```

Comment:
Think this one is for the delete index?

Reply:
Updated it to 'where x is smaller than all 3 digit numbers' since these are examples of incorrect delete commands